cmd/root.go

```diff
@@ -54,6 +55,13 @@ store the results of the check, use --store.`,
 
 		for _, result := range results {
 			fmt.Println(result)
+			if !result.Healthy {
+				allHealthy = false
+			}
+		}
+
+		if !allHealthy {
+			os.Exit(1)
```

Comment:
If I'm not wrong, it looks to me that this is about setting the exit code to 1 (rather than the default 0).